src/main/java/com/google/sps/data/MapsRequest.java

```diff
@@ -84,4 +114,11 @@ public static int getDurationFromDirectionsResult(DirectionsResult result) throw
     }
     return timeInSeconds;
   }
+
+  public static PlacesSearchResponse getLibraryLocations(LatLng location) throws ApiException, IOException, InterruptedException {
+    return PlacesApi.textSearchQuery(getGeoApiContext(), PlaceType.LIBRARY)
+    .location(location)
+    .radius(50000)
+    .await();
+  }
```

Comment:
Optional: it might be worth overloading this function so that you have one that allows you to set the radius, and one that defaults to `50000`.

This makes the code more flexible in the future.

```suggestion
public static PlacesSearchResponse getLibraryLocations(LatLng location) throws ApiException, IOException, InterruptedException {
  return getLibraryLocations(location, 50000);
}

public static PlacesSearchResponse getLibraryLocations(LatLng location, int radius) throws ApiException, IOException, InterruptedException {
    return PlacesApi.textSearchQuery(getGeoApiContext(), PlaceType.LIBRARY)
    .location(location)
    .radius(radius)
    .await();
```

Reply:
It makes sense, I made the change.